src/main/java/com/google/sps/servlets/course/NewClass.java

```diff
@@ -52,66 +52,54 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     try {
       String className = request.getParameter("className").trim();
 
-      // Prevents creating duplicate classes
```

Comment:
Why are duplicate classes allowed now, or is this being prevented elsewhere?

Reply:
In case the same course is offered at two different schools (like if cs101 is offered at both umd and duke), then we would let course owners take the same name. Since the only place the class name is used is to show users what class they're in, duplicate names won't affect anything.